Onboarding.md

```diff
@@ -0,0 +1,108 @@
+# Onboarding: ec-go-sdk
+
+## Purpose
+This document is a collection of all onboarding related information, tips & 
+tricks, etc. for first-time SDK contributers.
```

Comment:
This document is a collection of all `onboarding-related` information, tips & 
tricks, etc. for first-time SDK `contributors`.

Reply:
Fixed